src/helpers/notifications/Notify.js

```diff
@@ -0,0 +1,39 @@
+import models from '../../models';
+import sendMail from '../Mail/sendMail';
+import eventEmitter from './EventEmitter';
+
+const { Notification, Opt, User } = models;
+
+const notify = async (data) => {
+  let inAppNotification = {};
+  let emailNotification = {};
+
+  const { resource, user, inAppMessage, emailMessage } = data;
+  const optedin = await Opt.findAll({ where: { userId: user.followed } });
+  optedin.map(async (subscription) => {
+    const { dataValues } = await User.findOne({ where: { id: user.userId } });
+    switch (subscription.type) {
+      case 'email':
+        emailNotification = await Notification.create({ userId: user.userId,
+          resource,
+          message: emailMessage,
+          type: subscription.type });
+        await sendMail(dataValues.email, 'notification', { message: emailMessage });
+        break;
+      case 'inapp':
+        inAppNotification = await Notification.create({ userId: user.userId,
+          resource,
+          message: inAppMessage,
+          type: subscription.type });
+        eventEmitter.emit('new_inapp', inAppMessage, dataValues);
+        break;
+      default:
+        break;
+    }
+  });
+  const response = { inAppNotification,
```

Comment:
You can a return statement directly here instead of creating a variable that is taking up memory space.
`return { inAppNotification, emailNotification }`